services/common/src/redux/utils/helpers.ts

```diff
@@ -624,6 +624,9 @@ export const getMineReportStatusDescription = (
 ) => {
   const currentDate = moment().format("YYYY-MM-DD");
   const MINE_REPORT_STATUS_DESCRIPTION_HASH = {
+    [MINE_REPORT_SUBMISSION_CODES.NON]: `The ministry has requested for the report from the proponent through MineSpace. Requested by ${
```

Comment:
Is this text from the design?  The grammar feels awkward to me.  I would think it should be "The ministry has requested the report from the proponent through Minespace."

But if that's what the designers want, then 🤷🏻

Reply:
Yes, I saw it in figma. 
![image](https://github.com/bcgov/mds/assets/72251620/3b552af1-f49b-41bd-a202-d723edaaed04)